docs/reports/requirements/5_tech-stack.md

```diff
@@ -1,19 +1,80 @@
+tech stacks identified are outlined in the table below, and discussion and rational for our chosen tech stack (bolded)
+will follow.
+
+| Tech Stack               | Group Familiarity | Feasibile with selected approach | Easily integrates into tech stack                             | Quality Documentation Available |
+|--------------------------|-------------------|----------------------------------|---------------------------------------------------------------|---------------------------------|
+| **Node.Js + Typescript** | 2/4               | Yes                              | Yes - Aligns with Kukarella Tech & can be delivered as an API | Yes                             |
+| Django + Python          | 1/4               | Yes                              | Yes - Deliverable as an API                                   | Yes                             |
+| Laravel + PHP            | 1/4               | Yes                              | Yes - Deliverable as an API                                   | Yes                             |
+
+## Node.Js + Typescript
+
+### Pros:
+- Our group has more familiarity with Node and Typescript compared to the other potential approaches
+- The existing Kukarella tech stack is build using Node and Typescript. This means, if desired, our application could be
+  built into Kukarellas existing systems rather than being left as an external API.
+- Both Node and Typescript have outstanding documentation, as well as numerous resources for learning available online.
+
+### Cons:
+- Not all members of the group are familiar with Typescript (though all are familiar with Javascript).
+
+## Django + Python
+
+### Pros:
+- Python is very familiar to many of our group members
+- Lots of external libraries are available
+- Documentation is widely accessible
+### Cons:
+- Only one member of our group has experience with Django
+- Does not align with the Kukarella tech stack (though can be delivered as an API)
+
+## Laravel + PHP
+
+### Pros:
+- Laravel makes it very easy to spin up APIs
+
+### Cons:
+- Majority of our group is unfamiliar with PHP
+- Not as much documentation available for PHP
+
+For our implementation, we have decided to move forward in building our application using Node.Js and Typescript. We
+feel it makes the most sense to stick with what a majority of us have experience with, especially if it is the same tech
+stack that Kukarella uses within their own codebase. The reasoning behind our desire to maintaining one homogenous tech
+stack is that should our system require any maintenance in the future, it will be easier for Kukarella engineers to
+maintain provided they are already familiar with the tech it is built using.
+
+Given our current requirements, our application is purely a backend application. This means we do not currently expect
+to develop any front-end UI. Should future tasks be identified which require front-end development, we will use simple
```

Comment:
The "should future tasks be identified which require front-end development" is a little wordy. Perhaps something along the lines of, "If the client requests a front-end user interface"

Side note: this does raise questions of if the client can continue to request additional features throughout development and if we'll have time to do it all.

Reply:
I think the front-end requirement is coming from Gema not the client. Could re-word to be: 

"If a frontend is required for peer testing purposes..."

Reply:
Yeah - a good question to ask the prof potentially, though the client has mentioned that should there be additional time, then additional features would be good... Just covering all bases with the mention of front end stuff here